browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -305,9 +305,14 @@ class RewardsDOMHandler : public WebUIMessageHandler,
 
 namespace {
 
+const char kShouldShowPublisherAdsOnParticipatingSites[] =
+    "shouldShowPublisherAdsOnParticipatingSites";
+
 const char kShouldAllowAdConversionTracking[] =
     "shouldAllowAdConversionTracking";
 
+const char kTrue[] = "true";
+
```

Comment:
nit: Ah now, you know what I'm going to say... defining `kTrue` in multiple places... tsk... :D can we avoid it easily for this PR?

Reply:
Not at this time as out of scope of this PR, but we do need to look at sharing these values